big_tests/tests/sm_SUITE.erl

```diff
@@ -27,14 +28,16 @@
 all() ->
     [{group, parallel},
      {group, parallel_manual_ack_freq_1},
+     {group, stale_h},
      server_requests_ack_freq_2
      ].
 
 groups() ->
     G = [{parallel, [parallel], parallel_test_cases()},
          {parallel_manual_ack_freq_1, [parallel], parallel_manual_ack_test_cases()},
+         {stale_h, [], stale_h_test_cases()},
          {manual_ack_freq_long_session_timeout, [parallel], [preserve_order]}],
-    ct_helper:repeat_all_until_all_ok(G).
+    G.
```

Comment:
I may be not aware of this so I ask. Did we decide to stop using the `repeat_all_until_all_ok`?

Reply:
Oh damn, no, I just git-staged all instead of by patches. I personally dislike strongly that `repeat_all_until_all_ok`, first thing I do when I start working on any suite is _removing it_ and then just don't add it to the commit. This time I added it by mistake, but I'd love a discussion on why is that thing any good 🙁